tests/foreman/virtwho/test_api_virtwho.py

```diff
@@ -97,17 +121,15 @@ def test_positive_deploy_configure_by_id(self):
             (guest_name, 'product_id={} and type=STACK_DERIVED'.format(
                 self.vdc_physical))]
         for hostname, sku in hosts:
+            vdc_id = None
```

Comment:
Don't need to define `vdc_id = None` to here because the following if statements will always be covered.

Reply:
ok, thanks